tests/e2e/create/create_test.go

```diff
@@ -10,7 +10,7 @@ import (
 	"ocm.software/ocm/api/ocm"
 	"ocm.software/ocm/api/ocm/compdesc"
 	ocmv1 "ocm.software/ocm/api/ocm/compdesc/meta/v1"
-	"ocm.software/ocm/api/ocm/compdesc/versions/v2"
+	v2 "ocm.software/ocm/api/ocm/compdesc/versions/v2"
```

Comment:
to reviewer: my IDE keeps updating this one. Could you pls double check what happens on your side if you remove this?

Reply:
I think it's because `. "github.com/onsi/ginkgo/v2"`, you should use blank import for this package anyway, `_ "ocm.software/ocm/api/ocm/compdesc/versions/v2"`, it has many `init` function must be executed if you need to use ocm

Reply:
Not sure if I get this correctly. You suggest to import it like `_ "ocm.software/ocm/api/ocm/compdesc/versions/v2"`, correct? Not super familiar with it but reading up on it, it says: `This type of import is used to import a package solely for its side effects, without directly using any of its exported identifiers in the importing package.`. We use it however in line 219: `Expect(descriptor.SchemaVersion()).To(Equal(v2.SchemaVersion))`

Reply:
yes, basically with balnk import means the `init` function in the imported package will be automatically executed. In your case, the `v2` package contains a RegisterScheme function, however, if it's just need to compare this const `SchemaVersion`, then this blank import is not working.


```
package v2
const SchemaVersion = "v2"

func init() {
	compdesc.RegisterScheme(&DescriptorVersion{})
}

```

I haven't checked the usage tbh. I thought it's import for initialize some ocm features for testing, this implicit import used quite widely in this ocm package. But then, we have to import this `v2` package explicty. I tried to remove this v2 alias, it's not automatically added. I think it's your IDE problem.